src/components/ReviewInfoText/index.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components'
+import { Text } from '@gnosis.pm/safe-react-components'
+import { useSelector } from 'react-redux'
+
+import Paragraph from 'src/components/layout/Paragraph'
+import { currentSafe } from 'src/logic/safe/store/selectors'
+import { lg, sm } from 'src/theme/variables'
+import { TransactionFailTextProps, TransactionFees } from '../TransactionsFees'
+
+type ReviewInfoTextProps = TransactionFailTextProps & { safeNonce?: string; testId?: string }
+
+const ReviewInfoTextWrapper = styled.div`
+  background-color: ${({ theme }) => theme.colors.background};
+  padding: ${sm} ${lg};
+`
+
+export const ReviewInfoText = ({
+  gasCostFormatted,
+  isCreation,
+  isExecution,
+  isOffChainSignature,
+  safeNonce,
+  txEstimationExecutionStatus,
+  testId,
+}: ReviewInfoTextProps): React.ReactElement => {
+  const { nonce } = useSelector(currentSafe)
+  const transactionsToGo = parseInt(safeNonce || '', 10) - nonce
+
+  return (
+    <ReviewInfoTextWrapper data-testid={testId}>
```

Comment:
You're missing the styling that seems to have been applied everywhere else here.

Reply:
If we are talking about the 
> padding: ${sm} ${lg}

That's why I've passed that styling to the to the `ReviewInfoTextWrapper` styled component. 

Thats why I also removed some of the wrapping elements/classes across the Modals as they were repeating this same css property but dispersed across the codebase.

Reply:
No, I mean in reference to all of my other comments regarding classes.